scripts/gui/thrive_gui.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
What is the change on this line? Could you check that there isn't the wrong type of line change here.

Reply:
i didn't change nothing at that line, wrong type of line means that changes are in another line?

Reply:
The only reason I can think of that this line is marked as changed (as there is no visible change) is that the line end character has changed.